ui_framework/components/table/_table.scss

```diff
@@ -63,13 +63,22 @@ $tableCellPadding: 7px 8px 8px;
 }
 
 .kuiTableRow {
+  transition: background $globalAnimSlightResistance $globalAnimSpeedSlow;
   &:hover {
     .kuiTableRowHoverReveal {
       display: inline-block;
     }
   }
 }
 
+.kuiTableRow-isDeleted {
```

Comment:
Chatted with @snide off-PR. This class name will be changed to `kuiTableRow-isBeingDeleted` to reflect more accurately what this style will convey.